packages/cypress/src/steps.ts

```diff
@@ -30,6 +30,10 @@ function toRelativeTime(timestamp: string, startTime: number) {
   return toTime(timestamp) - startTime;
 }
 
+function toEventOrder(event: StepEvent) {
+  return ["test:start", "step:enqueue", "step:start", "step:end", "test:end"].indexOf(event.event);
```

Comment:
Tiny perf nit: an object lookup would be a lot faster here.

https://stackoverflow.com/questions/32234733/javascript-what-lookup-is-faster-array-indexof-vs-object-hash

But I know it does not matter, so feel free to ignore 🙃

Reply:
That tiny perf nit would lead to a tiny perf improvement ;)